Source/Core/BingMapsApi.js

```diff
@@ -15,48 +15,19 @@ define([
 
     /**
      * The default Bing Maps API key to use if one is not provided to the
-     * constructor of an object that uses the Bing Maps API.  If this property is undefined,
-     * Cesium's default key is used, which is only suitable for use early in development.
-     * Please generate your own key by visiting
```

Comment:
I think you should leave the information for how to generate their own bing key

Reply:
Good point.